docs/source/release-notes.rst

```diff
@@ -1,6 +1,12 @@
 Release Notes
 =============
 
+Version 0.16.3
+______________
+Engine:
+- Fix: Calling the `hopeit_server` command line without `--enabled_groups` parameter or with an empty one prevents to
```

Comment:
I think the parameter name is "enabled-groups" with hyphen, not underscore from the user point of view.

Reply:
Thank you. I fix it before the release